specification/cognitiveservices/data-plane/AnomalyDetector/preview/v2.0-preview/RootCauseAnalysis.json

```diff
@@ -0,0 +1,2143 @@
+          0.001,
+          1,
+          1000
+        ],
+        [
+          "US",
+          "__SUM__",
+          0.401,
+          401,
+          1000
+        ],
+        [
+          "JP",
+          "__SUM__",
+          0.01,
+          10,
+          1000
+        ],
+        [
+          "DE",
+          "__SUM__",
+          0.01,
+          10,
+          1000
+        ],
+        [
+          "CA",
+          "__SUM__",
+          0.01,
+          1,
+          100
+        ],
+        [
+          "__SUM__",
+          "__SUM__",
+          1,
+          1000,
+          1000
+        ]
+      ]
+    }
+  ],
+  "kind": "multidimensional"
+}
+    },
+    "MultiDimRCAUploadDataRequest": {
+      "required": [
+        "value"
+      ],
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/RootCauseAnalysisUploadDataRequest"
+        }
+      ],
+      "properties": {
+        "value": {
+          "description": "Multidimensional root cause analysis data",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/MultiDimRCADataPartition"
+          }
+        }
+      },
+      "x-ms-discriminator-value": "multidimensional"
+    },
+    "RootCauseAnalysisDataMeta": {
+      "required": [
+        "kind"
+      ],
+      "type": "object",
+      "properties": {
+        "kind": {
+          "description": "Root cause analysis kind",
+          "enum": [
+            "multidimensional"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "RootCauseAnalysisType",
+            "modelAsString": true
+          }
+        }
+      },
+      "discriminator": "kind"
+    },
+    "MultiDimRCADataMeta": {
+      "required": [
+        "dataSize",
+        "lastModifiedDateTime",
+        "partitionIdentity",
+        "timestamp"
+      ],
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/RootCauseAnalysisDataMeta"
+        }
+      ],
+      "properties": {
+        "partitionIdentity": {
+          "description": "Multidimensional root cause analysis partition identity",
+          "type": "string"
+        },
+        "dataSize": {
+          "format": "int32",
+          "description": "Multidimensional root cause analysis data size in the partition identity",
+          "type": "integer"
+        },
+        "timestamp": {
+          "format": "date-time",
+          "description": "Multidimensional root cause date time of the data",
+          "type": "string"
+        },
+        "lastModifiedDateTime": {
+          "format": "date-time",
+          "description": "Multidimensional root cause data last time updated",
+          "type": "string"
+        }
+      },
+      "x-ms-discriminator-value": "multidimensional"
+    },
+    "RootCauseAnalysisDataMetaListResponse": {
+      "required": [
+        "value"
+      ],
+      "type": "object",
+      "properties": {
+        "@nextLink": {
```

Comment:
All property names should be camel-case with no special characters.  Please use "nextLink".

Reply:
yes, updated in new PR